server/services/auth_service.go

```diff
@@ -100,3 +103,29 @@ func GetUserFromSession(c *fiber.Ctx) (*models.User, error) {
 
 	return user, nil
 }
+func InsertUserToSession(c *fiber.Ctx, user *models.User) error {
```

Comment:
```suggestion

func InsertUserToSession(c *fiber.Ctx, user *models.User) error {
```

Reply:
You can use GitHub to automatically apply my suggestion
 Anyway just add another line